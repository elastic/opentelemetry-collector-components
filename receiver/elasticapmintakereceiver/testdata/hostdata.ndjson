{"metadata": {"service":{"agent":{"name":"dotnet","version":"1.30.2-canary.0.4+3386ba62c316b311cf1da30278ce927a6735f2ce","activation_method":"nuget"},"environment":"Production","framework":{"name":"ASP.NET Core","version":"8.0.0.0"},"language":{"name":"C#"},"name":"SampleAspNetCoreApp","node":{},"runtime":{"name":".NET 8","version":"8.0.5"}},"process":{"pid":63039,"title":"SampleAspNetCor"},"system":{"detected_hostname":"gregs-mbp-2","hostname":"gregs-mbp-2"}}}
{"transaction": { "id": "945254c567a5417e", "trace_id": "0123456789abcdef0123456789abcdef", "parent_id": "abcdefabcdef01234567", "type": "request", "duration": 32.592981, "timestamp": 1496170407154000, "span_count": { "started": 43 }, "otel": { "attributes" : { "elastic.profiler_stack_trace_ids" : ["foo", "bar", "foo"]}}}}