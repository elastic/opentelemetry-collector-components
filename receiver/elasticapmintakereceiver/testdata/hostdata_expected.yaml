resourceSpans:
  - resource:
      attributes:
        - key: service.name
          value:
            stringValue: SampleAspNetCoreApp
        - key: service.version
          value:
            stringValue: ""
        - key: telemetry.sdk.language
          value:
            stringValue: dotnet
        - key: telemetry.sdk.name
          value:
            stringValue: ElasticAPM
        - key: deployment.environment
          value:
            stringValue: Production
        - key: deployment.environment.name
          value:
            stringValue: Production
        - key: process.pid
          value:
            intValue: "63039"
        - key: process.executable.name
          value:
            stringValue: SampleAspNetCor
        - key: agent.name
          value:
            stringValue: dotnet
        - key: agent.version
          value:
            stringValue: 1.30.2-canary.0.4+3386ba62c316b311cf1da30278ce927a6735f2ce
        - key: service.language.name
          value:
            stringValue: C#
    scopeSpans:
      - scope: {}
        spans:
          - attributes:
              - key: timestamp.us
                value:
                  intValue: "1496170407154000"
              - key: event.outcome
                value:
                  stringValue: success
              - key: processor.event
                value:
                  stringValue: transaction
              - key: transaction.id
                value:
                  stringValue: 945254c567a5417e
              - key: transaction.name
                value:
                  stringValue: ""
              - key: transaction.sampled
                value:
                  boolValue: true
              - key: transaction.root
                value:
                  boolValue: false
              - key: transaction.type
                value:
                  stringValue: request
              - key: transaction.result
                value:
                  stringValue: Success
              - key: transaction.duration.us
                value:
                  intValue: "32592"
            endTimeUnixNano: "1496170407186592981"
            parentSpanId: abcdefabcdef0123
            spanId: 945254c567a5417e
            startTimeUnixNano: "1496170407154000000"
            status:
              code: 1
            traceId: 0123456789abcdef0123456789abcdef
